include/emp/datastructs/BloomFilter.hpp

```diff
@@ -1,3 +1,11 @@
+/*** @note This file is included in Empirical (https://github.com/devosoft/Empirical) for convenience.
```

Comment:
Should we paste in the Empirical boilerplate header comments, too?

Reply:
I thought about it, but I didn't want to claim copyright. I tried to make this as analogous to the standard boilerplate as I could is there something specific it's missing?